resources/tests.mjs

```diff
@@ -3,19 +3,56 @@ import { BenchmarkTestStep } from "./benchmark-runner.mjs";
 const numberOfItemsToAdd = 100;
 export const Suites = [];
 
-Suites.enable = function (names) {
-    const lowerCaseNames = names.map((each) => each.toLowerCase());
-    this.forEach((suite) => {
-        if (lowerCaseNames.includes(suite.name.toLowerCase()))
-            suite.disabled = false;
-        else
-            suite.disabled = true;
-    });
+Suites.enable = function (names, tags) {
+    if (names?.length) {
+        const lowerCaseNames = names.map((each) => each.toLowerCase());
+        this.forEach((suite) => {
+            if (lowerCaseNames.includes(suite.name.toLowerCase()))
+                suite.disabled = false;
+            else
+                suite.disabled = true;
+        });
+    } else if (tags?.length) {
+        tags = new Set(tags);
+        tags.forEach((tag) => {
+            if (!Tags.has(tag))
+                console.error(`Unknown Suites tag: "${tag}"`);
+        });
+        this.forEach((suite) => {
+            suite.disabled = !suite.tags.some((tag) => tags.has(tag));
+        });
+    } else {
+        console.warn("Neither names nor tags provided. Enabling all default suites.");
+        this.forEach((suite) => {
+            suite.disabled = !("default" in suite.tags);
+        });
+        return;
+    }
+    if (Suites.some((suite) => !suite.disabled))
```

Comment:
Can we flip this condition sot that error case is in an early return if?
That'll make it easier to follow the code path.

Reply:
Not sure I fullly follow, did you mean?

```
if (this.every((suite) => suite.disabled) {
  ... throw errors...
 }
```

Reply:
Yes.

Reply:
DONE